README.md

```diff
@@ -35,5 +35,12 @@ A Persian (Farsi) Font
            url('Samim-Bold.ttf') format('truetype');
       font-weight: bold;
     }
+    
+    body {
+        /* optional enhancements for better rendering */
+        text-rendering: auto;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
These are not future and mobile friendly. 
It screw up rendering on HiDPI screens (most smart phones these days + Retina Display on Mac + potentially high end monitors and windows laptops)
I know it might still work for Desktops in Iran but I feel they shouldn't be recommended in Readme.